tests/init/jspdf.unit.spec.js

```diff
@@ -453,15 +453,21 @@ describe('jsPDF unit tests', () => {
 
     var writeArray = [];
     doc.__private__.setCustomOutputDestination(writeArray);
-    doc.__private__.setLineDash('1 1', 1);
+    doc.__private__.setLineDash('1 1', '1');

```

Comment:
Looking into the thing again, I wonder if we really should allow dashPhase to be a string. I do not see a use case for this. And the string would go directly to PDF ...

Maybe we could add an argument checking and require a number here as well.